src/app/guides/page.tsx

```diff
@@ -8,11 +8,18 @@ import CustomButton from "~/components/common/Button/button";
 import HBPCommentBox from "~/components/common/Comment";
 import { commentsData } from "~/components/common/Comment/ComentData";
 import CustomInput from "~/components/common/Input/input";
+import PaginatedTemplateList from "~/components/common/PaginatedTemplateList/PaginatedTemplateList";
 import Sidebar from "~/components/layouts/Sidebar";
+import templateTableList from "~/constants/templateList";
 import CharacterLimitTextarea from "../../components/common/CharacterLimitTextarea";
 
 const StyleGuide: React.FC = () => {
   const [text, setText] = useState("");
+  const [activePreview, setAcivePreview] = useState<number>(1);
+
+  const onPreview = (id: number) => {
+    setAcivePreview(id);
+  };
```

Comment:
Unnecessary func
Just pass directly to component like `onPreview={(id) => setActivePreview(id)} `